ProjectLighthouse.Tests/Unit/TicketTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using LBPUnion.ProjectLighthouse.Tickets;
+using LBPUnion.ProjectLighthouse.Tickets.Signature;
+using LBPUnion.ProjectLighthouse.Types.Users;
+using Xunit;
+
+namespace LBPUnion.ProjectLighthouse.Tests.Unit;
+
+[Trait("Category", "Unit")]
+public class TicketTests
+{
+    [Fact]
+    public void CanReadTicket()
+    {
+        TicketBuilder builder = new TicketBuilder().SetCountry("br")
+            .SetUserId(21)
+            .SetDomain("us")
+            .SetStatus(0)
+            .SetIssuerId(0x74657374)
```

Comment:
```suggestion
    public void CanReadTicket()
    {
        TicketBuilder builder = new TicketBuilder()
            .SetCountry("br")
            .SetUserId(21)
            .SetDomain("us")
            .SetStatus(0)
            .SetIssuerId(0x74657374)
```
Can this be moved down to be consistent with styling?

Reply:
I can, but any time the file is formatted Rider tries to move it back

Reply:
Ah, that's alright then.